docs/secret-rotation.md

```diff
@@ -0,0 +1,254 @@
+
+| The time interval in which FluxCD will check for these *drifts between configurations* is set in the cluster’s **helmrelease.yaml** found within the `fluxcd-config/components/<cluster-name>/` directory, under the `interval` variable. |
+| :---- |
+
+This should result in the **sealed secret** on the kubernetes cluster to contain the new secret.
+
+<a id="deleting-old-secrets-link"></a>
+###	**Deleting Old Secrets**
+
+Once the sealed secret has been updated on kubernetes it is time to do some *housekeeping* by now deleting the old credentials and unnecessary files created along the way, including, but not limited to, **old-credentials.yaml** and its corresponding application credential on Horizon.
+
+By following the steps outlined in **Create/Delete Credentials**, the option to delete application credentials can be found.
+
+Do note that in the scenario where the secret has been compromised, this should be the first step completed; this is because doing so invalidates the credential posing the security threat and prevents any operations using those credentials from being authenticated. Apart from this, and no longer needing to rename **credentials.yaml** to **old-credentials.yaml**, all other steps are the same.
+
+<a id="rotating-compromised-secrets-link"></a>
+###	**Rotating Compromised Secrets**
+As mentioned above, an application credential which is a security concern should be deleted before any other step; after which a new application credential, which will replace it, can be created.
+
+| Don’t forget to download the clouds.yaml file after creating the new application credential\! |
+| :---- |
+
+Due to having deleted the old application credentials there is no need to rename the **credentials.yaml** file and can instead be directly replaced with the template, plus the contents of the new application credential’s **clouds.yaml**.
+
+```
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <MAKE SURE THIS MATCHES “cloudCredentialsSecretName:” IN configmap.yaml>
+  namespace: capi-self
+  annotations:
+    # Allow the sealed secret controller to take over this secret after bootstrapping
+    sealedsecrets.bitnami.com/managed: "true"
+stringData:
+  clouds.yaml: |
+    clouds:
+      openstack:
+        auth:
+          auth_url: ...
+          application_credential_id: ...
+          application_credential_secret: ...
+        interface: ...
+        identity_api_version: 3
+        auth_type: ...
+```
+
+Then, much like the previous secret rotation, use Kubeseal to encrypt the new secret. This time round, the Kubeseal command below will seal the secret and overwrite **credentials.yaml** rather than create an intermediate **encrypted-creds.yaml** file.
+
+| kubeseal \--kubeconfig kubeconfig \--format yaml \--controller-name sealed-secrets \--controller-namespace sealed-secrets-system \--secret-file credentials.yaml  \--sealed-secret-file credentials.yaml |
+| :---- |
```

Comment:
Can we reformat code blocks / commands so that they're more readable please? E.g. wrap them in triple back ticks ``` and split them across multiple lines
```suggestion
kubeseal \
  --kubeconfig kubeconfig \
  --format yaml \
  --controller-name sealed-secrets \
  --controller-namespace sealed-secrets-system \
  --secret-file credentials.yaml  \
  --sealed-secret-file credentials.yaml
```
More details [here](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code). This also has the advantage that GitHub will render a little button on the page that lets you easily copy the code to your clipboard for copy/pasting.